precompiles/relay-encoder/RelayEncoder.sol

```diff
@@ -32,47 +30,43 @@ interface RelayEncoder {
     /// @custom:selector 813667a0
     /// @param amount: The extra amount to bond
     /// @return result The bytes associated with the encoded call
-    function encodeBondExtra(uint256 amount)
-        external
-        pure
-        returns (bytes memory result);
+    function encodeBondExtra(
+        uint256 amount
+    ) external pure returns (bytes memory result);
 
     /// @dev Encode 'unbond' relay call
     /// @custom:selector 51b14e57
     /// @param amount The amount to unbond
     /// @return result The bytes associated with the encoded call
-    function encodeUnbond(uint256 amount)
-        external
-        pure
-        returns (bytes memory result);
+    function encodeUnbond(
+        uint256 amount
+    ) external pure returns (bytes memory result);
 
     /// @dev Encode 'withdrawUnbonded' relay call
     /// @custom:selector d5ad108e
     /// @param slashes Weight hint, number of slashing spans
     /// @return result The bytes associated with the encoded call
-    function encodeWithdrawUnbonded(uint32 slashes)
-        external
-        pure
-        returns (bytes memory result);
+    function encodeWithdrawUnbonded(
+        uint32 slashes
+    ) external pure returns (bytes memory result);
 
     /// @dev Encode 'validate' relay call
     /// @custom:selector bb64ca0c
-    /// @param comission: Comission of the validator as partsPerBillion
+    /// @param commission: Commission of the validator as partsPerBillion
     /// @param blocked: Whether or not the validator is accepting more nominations
     /// @return result The bytes associated with the encoded call
-    function encodeValidate(uint256 comission, bool blocked)
-        external
-        pure
-        returns (bytes memory result);
+    function encodeValidate(
+        uint256 commission,
+        bool blocked
+    ) external pure returns (bytes memory result);
 
     /// @dev Encode 'nominate' relay call
     /// @custom:selector d2ea7b08
     /// @param nominees: An array of AccountIds corresponding to the accounts we will nominate
     /// @return result The bytes associated with the encoded call
-    function encodeNominate(uint256[] memory nominees)
-        external
-        pure
-        returns (bytes memory result);
+    function encodeNominate(
+        uint256[] memory nominees
```

Comment:
Shall we take the opportunity to make this bytes32? I think it represents better than uint256